Task05/include/Distance.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include "City.h"
+#include <cmath>
+#include <algorithm>
+#include <map>
+class Distance {
```

Comment:
you don't need a class for doing the calculations, you can save a map with lambda expression, and do the calculations